Lib/test/test_dict.py

```diff
@@ -1471,6 +1471,83 @@ def test_dict_items_result_gc(self):
         gc.collect()
         self.assertTrue(gc.is_tracked(next(it)))
 
+    def test_str_nonstr(self):
```

Comment:
Since this test depends on a CPython implementation detail,
```suggestion
    @support.cpython_only
    def test_str_nonstr(self):
```

Reply:
`@support.cpython_only` seems to be used on tests rely on cpython implementation details and may fail if another implementation is used. This test makes sure that a particular cpython implementation detail cannot be observed from python code so it should pass on all implementations.

Reply:
Thanks for the explanation! Btw, you don't have to force-push (the core dev will squash before merging). Committing normally without force-pushing makes the changes easier to track.